Fastlane/Fastfile

```diff
@@ -190,3 +190,27 @@ lane :pr_changes_contains_path do |options|
 
   changes_contains_path
 end
+
+lane :clean_up_release_from_tag do |options|
+  latest_tag = ENV['BITRISE_GIT_TAG']
+
+  latest_release_branch = "release/#{latest_tag}"
+  if `git ls-remote --heads origin #{latest_release_branch}`.empty?
+    puts "Branch #{latest_release_branch} doesn't exist. Nothing to delete."
+  else
+    sh "git push origin --delete #{latest_release_branch}"
+  end
+
+  if `git ls-remote --tags origin #{latest_tag}`.empty?
+    puts "Tag #{latest_tag} doesn't exist. Nothing to delete."
+  else
+    sh "git push origin --delete #{latest_tag}"
+  end
+end
+
+error do |lane, exception|
```

Comment:
I'm wondering if introducing the error handling here will cause issues in projects that include this Fastfile.
I just introduced it to the Fastfile on the Mule project (here: https://github.com/WeTransfer/Mule/pull/1740). The Mule projects Fastfile "includes" this Fastfile. I think it might result into a redundant definition, but I'm just guessing without having it tried.

Reply:
We do have a shared "error handling" function in `shared_lanes.rb`. I think it would make sense to add your code in there and remove this "error handling". What do you think?

Reply:
https://github.com/WeTransfer/WeTransfer-iOS-CI/pull/170/commits/3ce2a48166517831025be5ee52573ba17ee4a89f

Reply:
> I think it might result into a redundant definition. I think it might result into a redundant definition.

I thought we wouldn't have a problem with it, but after testing, we do have an issue. It's not major, but it could result in unexpected results.

<img width="1027" alt="image" src="https://user-images.githubusercontent.com/5387088/171014466-471a1774-effb-4ab0-81a8-a7e03a50506f.png">

Reply:
Dennis, I've removed the [error handling from the App's Fastfile](https://github.com/WeTransfer/Mule/pull/1752), leaving it only in the CI one.